### PR TITLE
#1228 - Update to Node 12.16

### PR DESCRIPTION
**Pull request by oderayi**
- Use node 12.16.o in CircleCI. 
- Updated dependencies

**Note**
----
I could not update central-services-shared due to breaking changes. This repo requires error handling framework to be worked in and have central-services-shared removed.

### CODE REVIEW

**Review by yosheeck — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-auth",
-  "version": "7.4.0",
+  "version": "9.1.0",
   "description": "Authentication module for central services",
   "main": "src/index.js",
   "scripts": {
```
Comment (rmothilal) [inline package.json:6]: please add dependency check and dependency update to the package.json and make sure that all dependencies are updated

**Review by rmothilal — CHANGES_REQUESTED** [review]
Please see comment

**Comment by rmothilal** [comment]
As per our conversation @oderayi will be updating the library to cater for the new error framework, this process will take some time

**Review by rmothilal — APPROVED** [review]
happy, this repo will bbe archieved

**Pull request merged**